import_export_celery/admin.py

```diff
@@ -9,6 +9,7 @@
 
 
 class JobWithStatusMixin:
+    @admin.display(description=_("Job status info"))
```

Comment:
Why was this removed?

Reply:
removed? I think you misread. I added this line with the wrapper to provide a verbose name for the custom admin propery of job_status_info :)

Reply:
Any updates on this @timthelion ? I would also benefit from such translations

Reply:
Aha, sorry, I did misread!